tests/Console/Command/SelfUpdateCommandTest.php

```diff
@@ -132,7 +132,7 @@ public function testExecute(
 
         $commandTester = $this->execute($command, $input, $decorated);
 
-        self::assertSame($expectedFileContents, file_get_contents($this->getToolPath()));
+        self::assertStringEqualsFile($this->getToolPath(), $expectedFileContents);
```

Comment:
:-1:
first value shall be expected value, not actual value.
you will get wrong error message in case of failure

Reply:
```
assertStringEqualsFile(string $expectedFile, string $actualString[, string $message = ''])
```
here we can't swap those, as one is content and the other a path to file

Reply:
yep, that;'s why this assertion is not used here, but simply `assertSame`.
we could use `assertFileEqualsString`, but such assertion doesn't exists.

Still, you proposed to use wrong assertion, as it assert different thing than we want. please revert that one